cmd/attestation-container/attestation-container.go

```diff
@@ -86,66 +97,94 @@ func (s *server) FetchAttestation(ctx context.Context, in *pb.FetchAttestationRe
 
 func validateFlags() {
 	if *platformCertificateServer != "" && *platformCertificateServer != "AMD" && *platformCertificateServer != "Azure" {
-		log.Fatalf("invalid --platform-certificate-server value %s (valid values: 'AMD', 'Azure')", *platformCertificateServer)
+		logrus.Fatalf("invalid --platform-certificate-server value %s (valid values: 'AMD', 'Azure')", *platformCertificateServer)
 	}
 }
 
 func main() {
 	flag.Parse()
+
+	// if logFile is not set, logrus defaults to stderr
+	if *logFile != "" {
+		// If the file doesn't exist, create it. If it exists, append to it.
+		file, err := os.OpenFile(*logFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0666)
+		if err != nil {
+			logrus.Fatal(err)
+		}
+		defer file.Close()
+		multi := io.MultiWriter(file, os.Stderr)
```

Comment:
I'm not sure I'm a fan of always enabling logging to stderr.  If a customer chooses to use a file, it should go there instead of stderr.  If you really want to allow multiple locations, then the customer should be able to explicitly list all output locations.